src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -73,6 +73,10 @@ public class HttpWebRequest : WebRequest, ISerializable
         private bool _preAuthenticate;
         private DecompressionMethods _automaticDecompression = HttpHandlerDefaults.DefaultAutomaticDecompression;
 
+        private static readonly object _syncRoot = new object();
```

Comment:
CoreFx coding guidelines say to use "s_" as prefx for static fields. So, this should be "s_syncRoot".

Reply:
Fixed.